src/main/java/io/jenkins/plugins/pipeline_cloudwatch_logs/CloudWatchRetriever.java

```diff
@@ -235,20 +232,9 @@ private void stream(OutputStream os, @CheckForNull String nodeId, @CheckForNull
     }
 
     private FilterLogEventsRequest createFilter() {
-        List<String> logStreamNames = new ArrayList<>();
-        String token = null;
-        do {
-            DescribeLogStreamsResult r = client.describeLogStreams(new DescribeLogStreamsRequest(logGroupName).withLogStreamNamePrefix(logStreamNameBase + "@").withNextToken(token));
-            for (LogStream ls : r.getLogStreams()) {
-                logStreamNames.add(ls.getLogStreamName());
```

Comment:
No longer necessary; `withLogStreamNamePrefix` on the `FilterLogEventsRequest` does this more simply.